src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -1720,7 +1720,7 @@
           short-summary: The OS Type. Linux or Windows. Windows not supported yet for "VirtualMachines" VM set type.
         - name: --os-sku
           type: string
-          short-summary: The os-sku of the agent node pool. Ubuntu or CBLMariner when os-type is Linux, default is Ubuntu if not set; Windows2019, Windows2022 or WindowsAnnual when os-type is Windows, the current default is Windows2022 if not set.
+          short-summary: The os-sku of the agent node pool. Ubuntu, CBLMariner or Ubuntu2204 when os-type is Linux, default is Ubuntu if not set; Windows2019, Windows2022 or WindowsAnnual when os-type is Windows, the current default is Windows2022 if not set.
```

Comment:
Is CBLMariner actually allowed here?

Maybe fix the docs?

Reply:
CBLMariner is no longer allowed: https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-rp?path=/resourceprovider/server/microsoft.com/containerservice/server/validation/validators/ossku/linux_validator.go&version=GBmaster&_a=contents

Henry, one of the leads for Mariner has confirmed that we should drop CBLMariner and replace it with AzureLinux in the docs instead.

updated the PR